src/Spryker/Shared/RabbitMq/Transfer/rabbitmq.transfer.xml

```diff
@@ -6,11 +6,23 @@
 
     <transfer name="QueueConnection">
         <property name="name" type="string" />
+
```

Comment:
why empty line?

Reply:
just to separate the "business" properties from the "required to create connection" properties